pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -58,7 +58,8 @@ func TestValidateKubeSchedulerConfiguration(t *testing.T) {
 				RetryPeriod:   metav1.Duration{Duration: 5 * time.Second},
 			},
 		},
-		BindTimeoutSeconds: &testTimeout,
+		BindTimeoutSeconds:       &testTimeout,
+		PercentageOfNodesToScore: 35,
```

Comment:
Curious about 35.

Reply:
Yes. Just a valid number (between 0 and 100) to test our validation.